src/lightning_app/core/app.py

```diff
@@ -51,6 +52,7 @@ def __init__(
         Arguments:
             root: The root LightningFlow component, that defined all
                 the app's nested components, running infinitely.
+            tutorial_mode: Whether to activate the event loop even when no external changes are received.
```

Comment:
Instead of introducing a `tutorial mode` flag, can we instead change the tutorial to force a counter increase when printing out? This will instruct users to form a correct mental picture of the execution model (e.g. "here we we increase the counter to force the loop to run, then we print out the result").

Alternatives to the counter:
- have a special method that forces the loop to rerun (I'm not in favor as it kind of works against the execution model and may be misleading for first time users)
- have a `log` or `print` method in flow that forces the loop to rerun (people may find it weird that using `self.print` in flow would have a different effect compared to just `print`)

Reply:
Wouldn't that trigger a state update, hence one loop run?